rudder-webapp/SPECS/rudder-webapp.spec

```diff
@@ -286,7 +286,6 @@ echo " Done"
 # Do this ONLY at first install
 if [ $1 -eq 1 ]
 then
-		echo -e '# This sources the configuration file needed by Rudder\n. /etc/sysconfig/rudder-apache' >> /etc/sysconfig/apache2
```

Comment:
OK, it's just that I was confused by the fact you  are removing this line in a non SLES specific section.
So we have created a useless file on redhat during the last few versions